### DISCUSSION TITLE
Web Interface not loading any content - Chrome Browser

### DISCUSSION THREAD

**Discussion by Lowaiz** [dani-garcia/vaultwarden#4252] [Q&A]
Hi,

Since 1.29, we are observing issues and problems with the loading of content in the web interface on Chrome/Chromium browsers.
Pretty much randomly, users with all the correct accesses can't load the content even with multiple refresh.
The only way to get the interface to load the content is to clear the cache, either by `ctrl+shift+R` the page or clearing all the browser cache.
So I'm supposing that it's a problem with the web vault but in order to dig all the possibilities, I'm opening this thread.

The bug happened once again with the version 1.30.1. I did a little debug session with one of the concerned users, but the js and network console of Chrome did not give any piece of useful information.

We are running Vaultwarden within Kubernetes.

Here is the debug string (without the config part, I don't think it is relevant, but ping me if you need any info):
### Your environment (Generated via diagnostics page)
* Vaultwarden version: v1.30.1
* Web-vault version: v2023.10.0
* OS/Arch: linux/x86_64
* Running within Docker: false (Base: Not applicable)
* Environment settings overridden: false
* Uses a reverse proxy: true
* IP Header check: true (X-Forwarded-For)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Browser/Server Time Check: true
* Server/NTP Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: PostgreSQL
* Database version: PostgreSQL 11.19 on x86_64-pc-linux-gnu, compiled by gcc (GCC) 7.3.1 20180712 (Red Hat 7.3.1-12), 64-bit
* Clients used: 
* Reverse proxy and version: 

Thanks you for any help you could bring !

**Comment by BlackDex** [comment]
I'm suspecting the reverse proxy here. It probably caches the old html/js/css files and causes issue in some way.

It could also be setting a browser cache expire which is very long.

**Reply by Lowaiz** [reply]
Hello,

First, thanks for the express response, it's really appreciated. 

Then, I checked our proxy and the config (nginx) doesn't have any config about cache, it's pretty much just a nginx server with a location `/` with a proxy pass.
For the browser setting, I have no clue what setting to search for. I explored the [policy list](https://docs.google.com/document/d/1g3KyF5I4eJ58sBGti7iV4O8Kgydb37aDY_0LjCYNqow/edit), and did not find any useful option. Do you have a idea of what I need to search for ?

**Reply by BlackDex** [reply]
I would suggest to double check the wiki in regards to the proxy configs: https://github.com/dani-garcia/vaultwarden/wiki/Proxy-examples Pay attention to all the fine details like having a `/` or not does make a huge difference.

Also, check your `DOMAIN` config which you set for Vaultwarden that is really matches what it needs to be for the outside world.
And also check your nginx logs.

I noticed that you are not running via docker, while that should still allow access to for example `/api/config`, it could be faulty configured for the web-vault.

**Reply by Lowaiz** [reply]
I don't think that there is a problem with the config on the server side. The instance works for almost any one, 99% of the time.
The bug only occurs with Chrome users. Firefox users, including myself, never had it. And it's occurring really randomly.
And with a `ctrl+shift+r` as a fix when it occurs, I can't think of a problem at server side.

**Reply by BlackDex** [reply]
If that is the case, then it might just be Chrome, or some extensions which cause issues.
I use Chrome sometimes to during testing, and i never experienced this behavior before.

Maybe Chrome caches stuff differently.
Try to run it with the developers tools open, or debug mode or something.
But i'm afraid we can't help with that.
